src/AstroPanda.Blazor.Toolkit/Services/PeristentStateService.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Blazored.LocalStorage;
 
-namespace AstroPanda.Blazor.Toolkit.Services;
+namespace AstroPanda.Blazor.Toolkit
```

Comment:
```suggestion
namespace AstroPanda.Blazor.Toolkit;
```
I think this is going to fail to build